commands/run.js

```diff
@@ -65,67 +68,101 @@ function terminateFunction (callback) {
   });
 }
 
+function callCommand (command, parameters, shouldForward, pid) {
+  let spawnCommand = spawn(command, parameters);
+  spawnCommand.stdout.on('data', (rawData) => {
```

Comment:
'data' means you received some data, it may not be all. If you're block buffered between processes, it'd be possible to get part of the chrome message in one callback, and the rest in another. Or unlikely, but even split across 3 or more reads.

Reply:
In theory this is possible, but I haven't encountered this yet in my testing.  So far all of the `data` chunks have been split at newlines